src/qmapshack/gis/rte/router/brouter/CRouterBRouterSetupWizard.cpp

```diff
@@ -18,12 +18,8 @@
 
 #include "gis/rte/router/brouter/CRouterBRouterSetupWizard.h"
 
-#include <JlCompress.h>
-
 #include <QFileDialog>
 #include <QMessageBox>
-#include <QNetworkReply>
-#include <QNetworkRequest>
 
 #include "CMainWindow.h"
 #include "gis/rte/router/brouter/CRouterBRouterSetup.h"
```

Comment:
#include "helpers/CWebPage.h" is not used directly. Remove it and fix any indirect dependencies.

Reply:
fixed